models/draft.js

```diff
@@ -1,30 +1,66 @@
 const mongoose = require('mongoose');
+
 const cardSchema = require('./cardSchema');
+const CURRENT_SCHEMA_VERSION = require('./migrations/draftMigrations').slice(-1)[0].version;
+
+// Details on each pack, how to draft and what's in it.
+const Pack = {
```

Comment:
I'd like to see this reflect my earlier comment about making a more flexible custom draft schema

Reply:
Done in #1798